darts/models/torch_forecasting_model.py

```diff
@@ -937,3 +938,21 @@ def _get_best_torch_device(self):
     def _get_learning_rate(self):
         for p in self.optimizer.param_groups:
             return p['lr']
+
+class TorchParametricProbabilisticForecastingModel(TorchForecastingModel, ABC):
```

Comment:
This seems like a good idea - could you also maybe adapt RNNModel so it uses it as well?

Reply:
Yes, I can, but firstly I want to figure out the problem with `output_dim` calculation that I mentioned in the PR comments.